README.md

```diff
@@ -68,3 +102,35 @@ sdk.rpc.chain.sendSignedTransaction(signedTx)
     ...
   });
 ```
+
+### Delegate stake tokens
+
+```js
+const sdk = ...
+const { createDelegateCCSTransaction } = require("codechain-stakeholder-sdk");
+
+// Delegate 100 tokens to tccq94guhkrfndnehnca06dlkxcfuq0gdlamvw9ga4f
+const tx = createDelegateCCSTransaction(sdk, "tccq94guhkrfndnehnca06dlkxcfuq0gdlamvw9ga4f", 100);
+const signedTx = tx.sign({ secret: "...", seq: "...", fee: "..." });
+sdk.rpc.chain.sendSignedTransaction(signedTx)
+  .then(txhash => {
+    // txhash: H256
+    ...
+  });
+```
+
+### Request to revoke a delegated stake tokens
```

Comment:
Is the article `a` necessary?

Reply:
'the', and 'token' would be appropriate. I'll fix them and merge this PR.